LibreNMS/Alert/Transport/Pagerduty.php

```diff
@@ -69,11 +70,11 @@ public function contactPagerduty($obj, $config)
         // EU service region
         if ($config['region'] == 'EU') {
             $url = 'https://events.eu.pagerduty.com/v2/enqueue';
-        }
-
-        // US service region
-        else {
+        } elseif ($config['region'] == 'US') {
+            // US service region
             $url = 'https://events.pagerduty.com/v2/enqueue';
+        } else {
+            $url = $config['custom-url'];
```

Comment:
I'd rewrite to check if 'custom-url' is set, elseif region is EU, else US.

then custom-url takes priority over (accidental) region settings